man/systemd-resolved.service.xml

```diff
@@ -409,7 +407,6 @@ search foobar.com barbar.com
       <citerefentry><refentrytitle>resolvectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry project='man-pages'><refentrytitle>resolv.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry project='man-pages'><refentrytitle>hosts</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>systemd.network</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd-networkd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
```

Comment:
If we're removing mentions of `systemd.network` then mentions of `systemd-networkd.service` should be dropped as well.

Reply:
Yes, there are a lot more issues as I've mentioned here: [RHEL-22167#comment-23844272](https://issues.redhat.com/browse/RHEL-22167?focusedId=23844272&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23844272).

As you mentioned, it is a pointless exercise, so I would only remove mentions of `systemd.network` and close the issue. @dtardon What do you think?

Reply:
I replied on the issue. Let's wait a bit...